sdk/keyvault/keyvault-keys/test/README.md

```diff
@@ -1,40 +1,37 @@
 
-## Unit tests
+You can use existing Azure resources for the live tests, or generate new ones by using our [New-TestResources.ps1](https://github.com/Azure/azure-sdk-for-js/blob/master/eng/common/TestResources/New-TestResources.ps1) script, which will use an [ARM template](https://github.com/Azure/azure-sdk-for-js/blob/master/sdk/keyvault/test-resources.json) that already has all of the the necessary configurations.
 
-To run unit tests, go to the root of this library (keyvault-certificates) and run `npm run unit-test`.
-This will run tests in a purely nodejs environment, and in a browser environment using [Karma](https://karma-runner.github.io/latest/index.html).
-These tests aim to reproduce the behavior of our library against remote endpoints through previously recorded HTTP request and responses.
-You won't need to have any environment configuration, nor internet access, for these tests to run.
+The Azure resource that is used by the tests in this project is:
 
-Keep in mind that if you modify the test files, the API shape of this library, or the underlying network calls, the recordings need to be re-generated.
-You can read more in the section: [Regenerating recordings](#regenerating-recordings).
+- An [Azure Key Vault](https://docs.microsoft.com/en-us/azure/key-vault/general/basic-concepts). Your Azure application needs to be assigned as the **owner** of this Azure Key Vault. The steps are provided [below](#AAD-based-authentication). 
 
-## Integration tests
+To run the live tests, you will also need to set the below environment variables:
 
-Our integration tests will run against the live resources, which are determined by the environment variables you provide.
-To run the integration tests, you will need to execute `npm run integration-test`.
+- `TEST_MODE`: Should have `live` assigned.
+- `AZURE_CLIENT_ID`: The client ID of an Azure Active Directory application.
+- `AZURE_CLIENT_SECRET`: The client secret of an Azure Active Directory application.
+- `AZURE_TENANT_ID`: The Tenant ID of your organization in Azure Active Directory.
+- `KEYVAULT_NAME`: The name of the KeyVault to use.
 
-The environment variables needed are:
+The live tests in this project will create, modify and delete [keys](https://docs.microsoft.com/en-us/azure/key-vault/keys/about-keys) inside of the provided Azure Key Vault.
 
-- `AZURE_CLIENT_ID`: The Client ID of your Azure account.
-- `AZURE_CLIENT_SECRET`: The secret of your Azure account.
-- `AZURE_TENANT_ID`: The Tenant ID of your Azure account.
-- `KEYVAULT_NAME`: The name of the Key Vault you want to run the tests against.
+## AAD based authentication
 
-Integration tests won't need updated recordings, and they also won't change previous recordings unless specified (see [Regenerating recordings](#regenerating-recordings)).
+Go through the following setup in order to correctly setup the AAD credentials for tests that require it.
 
-## Regenerating recordings
+### Register a new application in AAD
```

Comment:
Looks like tests in Event Hubs, Service Bus and App Config also need a application to be registered and update the env variables AZURE_CLIENT_ID, AZURE_CLIENT_SECRET, AZURE_TENANT_ID. So, we will need this section in the test readme files for these projects as well

Reply:
I can do this in this PR!

Reply:
I have pushed an update that includes this.